tests/rpmbuild.at

```diff
@@ -2438,6 +2438,22 @@ No hello.debug
 [ignore])
 RPMTEST_CLEANUP
 
+RPMTEST_SETUP_RW([rpmbuild multiple subpackages source_nevr])
+AT_KEYWORDS([build] [source_nevr])
+RPMTEST_CHECK([
+runroot rpmbuild --quiet \
+  -ba ${RPMDATA}/SPECS/test-subpackages-epoch.spec
+rpm -q --queryformat '%{NEVR} %{SOURCERPM} %{SOURCENEVR}\n' ${RPMTEST}/build/RPMS/*/test-*.rpm
```

Comment:
You can actually skip the build entirely here and just use rpmspec:
`rpmspec -q --qf "%{nevr} %{sourcenevr}\n" /data/SPECS/test-subpackages-epoch.spec`

Also, we'll want to test that source rpms do NOT get this tag:
`rpmspec -q --srpm --qf "%{nevr} %{sourcenevr}\n" /data/SPECS/test-subpackages-epoch.spec`